taskboard/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .interface import TaskBoard
+from .types import Task, TaskList
```

Comment:
I am surprised you are not getting Ruff / MyPy failures when you import it like this without using it?
I was under the impression that you need to do stuff like 
`from .types import Task as Task` to make MyPy shut up?

Reply:
you sure you have enabled the strictest set?
i constantly get this at work

Reply:
i've enabled strict in mypy (doesn't give any errors), but there's no strict mode in ruff (that i know of). i checked ruff rules, it would have displayed an error if i didn't re-export the imported types with __all__